db/schema.rb

```diff
@@ -64,6 +64,19 @@
     t.datetime "expires_at",            null: false
   end
 
+  create_table "bulk_email_jobs", force: :cascade do |t|
+    t.integer  "facility_id",     limit: 4,     null: false
+    t.string   "sender",          limit: 255,   null: false
```

Comment:
I feel like sender should probably be a direct reference to the user. We don't delete users, so there shouldn't be that worry.